bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -449,6 +449,10 @@ <h2 class="primary go">{{ _('Help Mozilla users') }}</h2>
   <button id="scroll-prompt">{{ _('Discover more') }}</button>
 </main>
 
+{% if waffle.switch('fundraiser-home-takeover') and l10n_has_tag('eoy_fundraising_2015') %}
```

Comment:
I'm talking about the strings in the tile (that's where we're currently using that tag).

The proposed tag works for me.